src/inputs/Table/Table.js

```diff
@@ -412,6 +503,43 @@ Table.propTypes = {
    * If not defined, this button will not be rendered.
    */
   onExport: PropTypes.func,
+  /*
+   * Callback function that will be called when user click on the add row button.
+   * If not defined, this button will not be rendered.
+   */
+  onAddRow: PropTypes.func,
+  /*
+   * Callback function that will be called when user click on the delete row button.
+   * If not defined, this button will not be rendered.
+   */
+  onDeleteRow: PropTypes.func,
+  /*
+   * Function to update the label of the delete confirmation dialog body
+   */
+  buildDialogBodyLabel: PropTypes.func.isRequired,
+};
+
+const DEFAULT_LABELS = {
+  clearErrors: 'Clear',
+  label: PropTypes.string,
+  loading: 'Loading...',
+  errorsPanelMainError: 'File load failed.',
+  placeholderTitle: 'Import your first data',
+  placeholderBody: 'After importing a valid csv or xlsx file, your data will be displayed in an interactive table.',
+  import: 'Import',
+  export: 'Export',
+  addRow: 'Add a new row',
+  deleteRows: {
+    tooltip: 'Remove selected rows',
+    dialog: {
+      title: 'Delete selected lines?',
+      body: () => 'Are you sure you want to delete these lines?',
+      cancel: 'Cancel',
+      confirm: 'Delete',
+      checkbox: "Don't show this message again",
+    },
+  },
+  fullscreen: PropTypes.string,
```

Comment:
Something's wrong with the two values `PropTypes.string` :slightly_smiling_face:

Reply:
DONE